docs/packages/pkg/chaiscript.rst

```diff
@@ -0,0 +1,39 @@
+.. spelling::
+
+    chaiscript
+
+.. index:: scripting ; chaiscript
+
+.. _pkg.chaiscript:
+
+ChaiScript
+==========
+
+-  `Official <https://chaiscript.com>`__
+-  `Offical Github: <https://github.com/ChaiScript/ChaiScript>`__
+-  `Hunterized <https://github.com/hunter-packages/chaiscript>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/chaiscript/CMakeLists.txt>`__
+-  Added by `Chris Kaminski <https://github.com/ckaminski>`__ (`pr-1351 <https://github.com/ruslo/hunter/pull/1351>`__)
+
+.. literalinclude:: /../examples/chaiscript/CMakeLists.txt
+  :language: cmake
+  :start-after: # DOCUMENTATION_START {
+  :end-before: # DOCUMENTATION_END }
+
+
+Requirements
+============
+- C++14 
+- Boost : Threads 
```

Comment:
Installation of Boost will be done by Hunter automatically, I don't see the point of mentioning it explicitly.